Packs/OTRS/Integrations/OTRS/OTRS.py

```diff
@@ -500,21 +497,17 @@ def close_ticket_command():
 
 def update_ticket(ticket_id, title=None, queue=None, state=None, priority=None,
                   article=None, ticket_type=None, df=None, attachments=None):
-
     kwargs = {'Type': ticket_type}
-
-    response = client.ticket_update(
-        ticket_id,
-        Title=title,
-        Queue=queue,
-        State=state,
-        Priority=priority,
-        article=article,
-        dynamic_fields=df,
-        attachments=attachments,
-        **kwargs
-    )
-    return response
+    args = {'ticket_id': ticket_id,
+            'Title': title,
+            'Queue': queue,
+            'State': state,
+            'Priority': priority,
+            'article': article,
+            'dynamic_fields': df,
+            'attachments': attachments,
+            'kwargs': kwargs}
+    return execute_otrs_method('ticket_update', args)
```

Comment:
samre for the rest of the methods
```suggestion
    return execute_otrs_method(client.ticket_update, args)
```